### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by justcomplaining**
In the current state of the website and the mass of missing strings, I am not sure whether it is even sensible to merge new translations, but here it is.

### CODE REVIEW

**Comment by tfmorris** [comment]
Thanks for the contribution!

There is work in progress to update the missing I18N strings on the branch at #2442
Specifically this file can be used as a better starting point: https://github.com/internetarchive/openlibrary/blob/31bcaa079548cfb6a8e42c7ff295dd91ff19ddbc/openlibrary/i18n/messages.pot 

It might be better to wait until that branch is wrapped up and merged (or at least rebased) though, so that you can both get the new starting template and the updated HTML to test your translations against.

**Comment by justcomplaining** [comment]
> It might be better to wait until that branch is wrapped up and merged (or at least rebased) though, so that you can both get the new starting template and the updated HTML to test your translations against.

As I have no experience with the whole process whatsoever: is it possible to somehow update the old translation file with a new version of the template so existing translations won't be lost?

**Comment by tfmorris** [comment]
> is it possible to somehow update the old translation file with a new version of the template so existing translations won't be lost?

Yes, the tooling supports that, although I'm still gaining experience with how well it works in practice. For things that are unchanged, it should be pretty good. Where the source msgid changed, it'll attempt to find a close match, which doesn't always work, but it does retain the old msgid/msgstr pair as a comment so that it can be reused for manual matching.

BTW, I just noticed that the pull request is from your `master` branch. We recommend creating a feature branch and doing the pull request from that, so you don't accidentally pollute your pull request with unintended commits.

File: openlibrary/i18n/de/messages.po [hunk 1]
```diff
@@ -0,0 +1,2327 @@
...
+
+#: notfound.html:13 openlibrary/templates/work_search.html:129
+msgid "Try something else?"
+msgstr "Etwas anderes probieren?"
+
+#: openlibrary/templates/subjects.html:15 view.html:13
+#, python-format
+msgid "%s work"
+msgid_plural "%s works"
+msgstr[0] "&s Werk"
```
Comment (tfmorris) [inline openlibrary/i18n/de/messages.po]: Should be %s, not &s

File: openlibrary/i18n/de/messages.po [hunk 2]
```diff
@@ -0,0 +1,2327 @@
...
+msgid "History"
+msgstr "Verlauf"
+
+#: history.html:18
+#, python-format
+msgid "Created %s"
+msgstr "%s erstellt"
+
+#: history.html:25
+msgid "revisio
```
Comment (tfmorris) [inline openlibrary/i18n/de/messages.po]: msgid `Revision` ?

**Comment by tfmorris** [comment]
Thanks for your contribution!

**Comment by justcomplaining** [comment]
> BTW, I just noticed that the pull request is from your `master` branch. We recommend creating a feature branch and doing the pull request from that, so you don't accidentally pollute your pull request with unintended commits.

Thanks for the help, I'll look into how all that works before my next pull request :)

**Pull request merged**